app/app.py

```diff
@@ -0,0 +1,49 @@
+from flask import Flask, render_template
+from flask_restful import Api
+import logging
+import os
+import glob
+import sys
+import traceback
+
+logging.basicConfig(level=logging.INFO)
+STATIC_DIR = os.path.join(os.path.dirname(
+    os.path.abspath(__file__)), 'static', 'gen')
+
+app = Flask(__name__,
+            static_folder='static',
+            static_url_path='/static')
+
+
+@app.route('/docs')
+def docs():
+    return render_template('docs.html')
+
+
+@app.route('/')
+def index():
+    css_files = glob.glob(os.path.join(STATIC_DIR, '*.css'))
+    css = [os.path.split(item)[1] for item in css_files]
+    js_files = glob.glob(os.path.join(STATIC_DIR, '*.js'))
+    js = [os.path.split(item)[1] for item in js_files]
```

Comment:
This loads all of the css/js files (respectively) that exist in the static directory (defined above) and puts them into a list. That list is passed to the view so that the view just added the script tags programatically. As grunt generates new filenames on change, hardcoding those paths is not feasible.